authorizations/authorization-ranger/src/test/java/org/apache/gravitino/authorization/ranger/integration/test/RangerFilesetIT.java

```diff
@@ -0,0 +1,536 @@
+                            {
+                              set("fs.defaultFS", defaultBaseLocation());
+                            }
+                          });
+                  Assertions.assertThrows(
+                      Exception.class,
+                      () ->
+                          userFileSystem.listFiles(new Path(storageLocation(filenameRole)), false));
+                  Assertions.assertThrows(
+                      Exception.class,
+                      () ->
+                          userFileSystem.mkdirs(
+                              new Path(
+                                  String.format("%s/%s", storageLocation(filenameRole), "test4"))));
+                  userFileSystem.close();
+                  return null;
+                });
+
+    catalog.asFilesetCatalog().dropFileset(NameIdentifier.of(schemaName, fileset.name()));
+  }
+
+  private void createCatalogAndSchema() {
+    GravitinoMetalake[] gravitinoMetalakes = client.listMetalakes();
+    Assertions.assertEquals(0, gravitinoMetalakes.length);
+
+    client.createMetalake(metalakeName, "comment", Collections.emptyMap());
+    metalake = client.loadMetalake(metalakeName);
+    Assertions.assertEquals(metalakeName, metalake.name());
+
+    metalake.createCatalog(
+        catalogName,
+        Catalog.Type.FILESET,
+        provider,
+        "comment",
+        ImmutableMap.of(
+            IMPERSONATION_ENABLE,
+            "true",
+            AUTHORIZATION_PROVIDER,
+            "ranger",
+            RANGER_SERVICE_NAME,
+            RangerITEnv.RANGER_HDFS_REPO_NAME,
+            AuthorizationPropertiesMeta.RANGER_ADMIN_URL,
+            RANGER_ADMIN_URL,
+            RANGER_AUTH_TYPE,
+            RangerContainer.authType,
+            RANGER_USERNAME,
+            RangerContainer.rangerUserName,
+            RANGER_PASSWORD,
+            RangerContainer.rangerPassword));
+
+    catalog = metalake.loadCatalog(catalogName);
+    catalog
+        .asSchemas()
+        .createSchema(schemaName, "comment", ImmutableMap.of("location", defaultBaseLocation()));
+    Schema loadSchema = catalog.asSchemas().loadSchema(schemaName);
+    Assertions.assertEquals(schemaName, loadSchema.name());
+    Assertions.assertNotNull(loadSchema.properties().get("location"));
+  }
+
+  private String defaultBaseLocation() {
+    if (defaultBaseLocation == null) {
+      defaultBaseLocation =
+          String.format(
+              "hdfs://%s:%d/user/hadoop/%s",
+              containerSuite.getHiveRangerContainer().getContainerIpAddress(),
+              HiveContainer.HDFS_DEFAULTFS_PORT,
+              schemaName.toLowerCase());
+    }
+    return defaultBaseLocation;
+  }
+
+  private Fileset createFileset(String filesetName, Fileset.Type type, String storageLocation) {
+    return catalog
+        .asFilesetCatalog()
+        .createFileset(
+            NameIdentifier.of(schemaName, filesetName), "comment", type, storageLocation, null);
+  }
```

Comment:
If we didn't use this function, please remove it.

Reply:
done